### PR TITLE
Add TSC Meeting notes 05-04-2020

### PR DESCRIPTION
**Pull request by michdolan**
Signed-off-by: Michael Dolan <michdolan@gmail.com>

### CODE REVIEW

File: docs/tsc/meetings/2020-05-04.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      When CI moved to GH Actions, the bindings are compiling but not running. 
+      Unit tests are needed to make sure it is working.
+    - Doug: We also need some new good first issues for the project. There 
+      aren't many good first issues there now. Python unit tests could be 
+      included.
+
+* Built-in transforms
+    - Doug: More built-in transforms will be needed, and could be contributed 
+      by the community once the current PR #1006 is merged.
+    - Mark: I was hoping to have OpenColorMath in before going to far with
```
Comment (hodoulp) [inline docs/tsc/meetings/2020-05-04.md:114]: typo 'going to far' -> 'going too far'

**Comment by KelSolaar** [comment]
Been useful to have been around for this one :)

File: docs/tsc/meetings/2020-05-04.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+      repo.
+    - Joseph: I will reach out to them via email.
+    - Sean: The tricky part will be what to do with old stuff. Also the 
+      particulars of the config. But path forward is clear. It would be 
+      worthwhile to investigate a chair to oversee config maintenance and to 
+      have a clear owner - whoever has the time and detail.
+    - Mark: That person could also own the numbers in the config, which 
+      sometimes are debated.
+    - Doug: From ACES, Thomas Mansencal and Michael Parsons were identified as 
+      owners. ACES is encouraging OCIO to take ownership and become a product
```
Comment (doug-walker) [inline docs/tsc/meetings/2020-05-04.md:55]: Please replace the sentence "ACES is encouraging ..." with "I think ACES would like OCIO to take a bigger role in the creation and maintenance of the config, similar to a product partner."

Just wanted to clarify that I was stating my interpretation of the comments made at the ACES meeting and it is not an official request from the Academy to OCIO.

Comment (doug-walker) [reply docs/tsc/meetings/2020-05-04.md:55]: Also, I hope Thomas and Michael will continue their efforts on the config.  It is greatly appreciated!

Comment (doug-walker) [reply docs/tsc/meetings/2020-05-04.md:55]: @KelSolaar , please take note of my comments above, which I hope more accurately reflects what was said at the Monday meeting.  As was said at the Friday meeting, I think everyone would like you to continue the extremely valuable role you have been playing.  The Monday meeting was just trying to recap that for the OCIO TSC.  Just wanted to make sure there was no confusion.  ;)

Comment (KelSolaar) [reply docs/tsc/meetings/2020-05-04.md:55]: Thanks @doug-walker, time permitting I would be keen to continue indeed and timezone maybe participate in some of the TSC meetings.

Comment (michdolan) [reply docs/tsc/meetings/2020-05-04.md]: Fixed

File: docs/tsc/meetings/2020-05-04.md (hunk 1 above)
Comment (michdolan) [reply docs/tsc/meetings/2020-05-04.md]: Fixed

**Pull request merged**